components/brave_rewards/browser/rewards_service_browsertest.cc

```diff
@@ -376,29 +434,34 @@ class BraveRewardsBrowserTest :
         &brave_test_resp::surveyor_voting_credential_));
   }
 
-  void UpdateTestData() {
-    base::ScopedAllowBlockingForTesting allow_blocking;
-    base::FilePath path;
-    GetTestDataDir(&path);
-    if (grant_finished_) {
-      if (contribution_made_) {
-        ASSERT_TRUE(base::ReadFileToString(
-            path.AppendASCII("wallet_balance_contributed_resp.json"),
-            &brave_test_resp::wallet_));
-      } else if (tip_made) {
-        ASSERT_TRUE(base::ReadFileToString(
-            path.AppendASCII("wallet_balance_tipped_resp.json"),
-            &brave_test_resp::wallet_));
+  void UpdateContributionBalance(double amount, bool verified = false) {
+    if (verified) {
+        balance_ -= amount;
       } else {
-        ASSERT_TRUE(base::ReadFileToString(
-            path.AppendASCII("wallet_balance_funded_resp.json"),
-            &brave_test_resp::wallet_));
+        pending_balance_ += amount;
       }
-    } else {
-      ASSERT_TRUE(base::ReadFileToString(
-          path.AppendASCII("wallet_balance_empty_resp.json"),
-          &brave_test_resp::wallet_));
+  }
+
+  std::string BalanceDoubleToString(double amount) {
+    std::vector<std::string> parts = base::SplitString(
```

Comment:
Might be simpler to just use `base::StringPrintf` here.

Reply:
done